PrimeNumbers/PrimeNumbers.cpp

```diff
@@ -1,23 +1,90 @@
 #include "PrimeNumbers.h"
+#include <sstream>
 
 using namespace ::std;
 
 PrimeNumbers::PrimeNumbers()
 {
-	m_PrimeNumbers.reserve(3);
+	m_PrimeNumbers.push_back(m_smallestPrime);
 }
 
-PrimeNumbers::~PrimeNumbers()
+bool PrimeNumbers::IsPrimeNumber(const u64 number)
 {
-	m_PrimeNumbers.clear();
+	for (int i = m_smallestPrime; i < number; i++)
+	{
+		if (number %i == NULL)
```

Comment:
Please don't compare a number with NULL. It technically works but semantically, `NULL` is supposed to mean a nullpointer, not a name for the number zero.